packages/web/src/components/CheckboxField/_theme.scss

```diff
@@ -0,0 +1,24 @@
+@use '@tokens' as tokens;
+
+$spacing: tokens.$space-300;
+
+$label-typography: tokens.$body-medium-text-regular;
+$label-color-default: tokens.$text-secondary-default;
+$label-color-disabled: tokens.$text-secondary-disabled;
+$label-required-color: tokens.$emotion-danger-default;
+
+$input-width: 18px;
+$input-typography: tokens.$body-medium-text-regular;
+$input-border-color-error: tokens.$emotion-danger-default;
```

Comment:
I think these two (11 and 12) are not used.

Reply:
Fixed.